src/test/scala/com/virtuslab/scalaworkshop/intro/IntroSpec.scala

```diff
@@ -42,7 +42,7 @@ class IntroSpec extends WordSpec with Matchers {
         val value = throw new IllegalStateException()
         fail("Exception should have been thrown")
       } catch {
-        case ex => println("I am done")
+        case ex: Throwable => println("I am done")
```

Comment:
Consider using [`NonFatal`](https://www.scala-lang.org/api/current/scala/util/control/NonFatal$.html).

Reply:
I know. But it's not my example and, I think, Paweł wanted to have this code as simple as possible (without having to explain what NonFatal means).

Reply:
@pdolega Correct me if I'm wrong.

Reply:
Then just use `IllegalStateException`, as it doesn't introduce new entities. By the way, expecting  a broad variety of exceptions is IMO an antipattern.

Reply:
I'd say @odisseus is right. Can change it today in the evening.

Reply:
I know all of this...